tests/storage_tests/devices_test/stratis_test.py

```
@@ -375,7 +375,8 @@ def test_stratis_pool_start_stop_encrypted(self):
         blivet.partitioning.do_partitioning(self.storage)
 
         pool = self.storage.new_stratis_pool(name="blivetTestPool", parents=[bd],
-                                             encrypted=True, passphrase="fipsneeds8chars")
+                                             encrypted=True, passphrase="fipsneeds8chars",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "passphrase"

(S106)